docs/source/BPs/wireless_management.rst

```diff
@@ -30,46 +30,44 @@ Mappings to CIS Controls and Safeguards
 Actions
 **********************************************
 
+For |bp_title|, the necessary actions vary by maturity as detailed below.
+
 |Maturity1| Maturity
 &&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&
 
 For those organizations operating at a |Maturity1| maturity, the important thing is to keep it simple. Avoid using wireless in risky scenarios, such as transmitting election results without the technical support of a state agency or other technical body providing guidance.
 
-#. Leverage the Advanced Encryption Standard (AES) to Encrypt Wireless Data
-#. Create Separate Wireless Network for Personal and Untrusted Devices
+#. Leverage the advanced encryption standard (AES) to encrypt wireless data.
+#. Create separate wireless network for personal and untrusted devices.
+#. Change administrator passwords on routers and other wireless access points to a secure passphrase.
+#. Change the default access passphrase for wireless networks regularly, or enable user level authentication for private networks.
+#. Don't permit visitors to use your primary wireless network. Instead set up a separate network or enable a guest network.
+#. Carefully decide whether a new device will be allowed on the network; you don't need to permit every new device onto the network.
+#. Keep firmware and software up to date by including your router and other access points in your :doc:`patching <software_updates>` schedule.
+#. Use :doc:`the encryption best practice <encryptdataatrest>` for all network traffic.
```

Comment:
I don't think we want to talk about encryption at rest for network traffic @mike-garcia! sorry man!